airflow/providers/apache/hive/operators/hive_stats.py

```diff
@@ -82,15 +81,6 @@ def __init__(
         col_blacklist: list[str] | None | ArgNotSet = NOTSET,
         **kwargs: Any,
     ) -> None:
-        if col_blacklist is not NOTSET:
```

Comment:
You also need to remove `col_blacklist` argument and I guess fix some tests after that

Reply:
I guess not even fix rather than remove some tests because if the argument not exists, then there is no reason to keep them

Reply:
Yeah, no point, removing them